src/sentry/models/user.py

```diff
@@ -449,14 +452,14 @@ def do_write():
 
             self.save(force_insert=True)
 
-            # TODO(getsentry/team-ospo#190): the following is an RPC call which could fail for
-            # transient reasons (network etc). How do we handle that?
-            lost_password_hash_service.get_or_create(user_id=self.id)
+            if scope != ImportScope.Global:
```

Comment:
is it worth writing a test for this scenario?

Reply:
This is uncovered because I haven't thought of a good way to make these tests non-flaky atm - they trigger an async call, whereas the rest of the test is sync. I think this is okay for now though, I'll circle back and add coverage, and there's a TODO calling this out in the meantime.